sdk/storage/Azure.Storage.Common/src/StorageTransferOptions.cs

```diff
@@ -21,6 +21,11 @@ public struct StorageTransferOptions : IEquatable<StorageTransferOptions>
         /// </summary>
         public int? MaximumConcurrency { get; set; }
 
+        /// <summary>
+        /// The size of the first range requested (which can be larger than the other ranges).
```

Comment:
Can we maybe say `The size of the first range request in bytes.  Blobs smaller than this limit will be downloaded in a single request.  Blobs larger than this limit will continue being downloaded in chunks of size MaximumTransferLength.` or similar to give more of a hint about how to use.

Reply:
fixed.